substrate/frame/referenda/src/lib.rs

```diff
@@ -1304,6 +1309,11 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		}
 	}
 
+	#[cfg(any(feature = "std", test))]
+	fn do_integrity_test() {
+		T::Tracks::check_integrity().expect("Static tracks configuration is valid.");
```

Comment:
The expect could use a better message to tell the user what the problem is.

Reply:
Changed to return `&'static str` in the `check_integrity` for a proper error message.  
Since here we dont actually know what the issue was.

Reply:
Nice! I had the same idea while writing this comment, but just returning the error is a good solution :D